src/lib/utils.ts

```diff
@@ -75,13 +75,13 @@ let snapshotHeight = -1;
 
 export const createSnapshot = async (aeSdk) => {
   snapshotHeight = await aeSdk.getHeight();
-  await awaitKeyBlocks(aeSdk, 1);
```

Comment:
don't remember why those were needed

Reply:
mhh, me neither. did you check if removing this has any negative effect?

Reply:
I tried but couldn't see anything problematic